src/SiteManager.php

```diff
@@ -254,13 +265,6 @@ public function filterSitesFromGroups(array $groups) {
     return $sites;
   }
 
-  /**
-   * {@inheritdoc}
-   */
-  public function getSitesFromContent(NodeInterface $node) {
```

Comment:
@matthijs-va, dit is een breaking change?

Reply:
Is een deprecated methode uit de interface, ik heb verwijderd omdat phpcs de deprecation maar niks vond en een bepaalde formatting vroeg.